sagemaker_model_monitor/data_quality/nlp/data/validation.json

```diff
@@ -0,0 +1,727 @@
+{"label":1,"sentence":"His book"}
```

Comment:
What is the source of the train/test/val JSON data files? Let's aim to host them in S3 rather than committing directly to the repo.

Reply:
removed data files from the repo